docs/markdown/Reference-tables.md

```diff
@@ -377,6 +377,25 @@ machine](#Environment-variables-per-machine) section for details.
 *The old environment variables are still supported, but are deprecated
 and will be removed in a future version of Meson.*
 
+*changed in 1.3.0* Paths with spaces were split unconditionally to extract
+the [path to Ccache](Feature-autodetection.md#ccache). This broke passing
```

Comment:
"to extract the path to ccache, intrinsic compiler flags such as -m32 or --target, etc."